### PR TITLE
Check if dtypes is sorted as in ADLS file

### PR DESCRIPTION
**Pull request by Diego-H-S**
## Summary
Before adding this new feature, if the user passes a non-sorted dtypes argument like in the ADLS file, it creates an unsorted data frame to upload to Azure SQL table.
Also added more descriptions to azure_sql.py about how the credentials are loaded.

## Importance
Not relevant right now, because all our flows are well built but it would be important for any user that is not aware of the problem.

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [x] updates docstrings for any new functions or function arguments (if appropriate)
- [x] updates `CHANGELOG.md` with a summary of the changes

### CODE REVIEW

File: viadot/flows/adls_to_azure_sql.py [hunk 1]
```diff
@@ -5,8 +5,10 @@
 import pandas as pd
 from prefect import Flow, task
 from prefect.backend import get_key_value
+from prefect.engine import signals
 from prefect.utilities import logging
 
+
```
Comment (Rafalz13) [inline viadot/flows/adls_to_azure_sql.py:11]: delete extra line

**Review by Rafalz13 — CHANGES_REQUESTED** [review]
Please add tests to the new functionality.

File: tests/integration/flows/test_adls_to_azure_sql.py [hunk 2]
```diff
@@ -67,3 +70,35 @@ def test_df_to_csv_task_none(caplog):
...
+    }
+    task = check_dtypes_sort
+    n_dtypes = task.run(df=df, dtypes=dtypes)
+    assert list(dtypes.keys()) == list(n_dtypes.keys())
+
+    dtypes = {
+        "col2": "varchar(6)",
+        "col1": "varchar(6)",
+    }
+    task = check_dtypes_sort
```
Comment (Rafalz13) [inline tests/integration/flows/test_adls_to_azure_sql.py:91]: Do you have to overwrite the task object every time? Maybe once on line 83, then just  run it.

Comment (Diego-H-S) [reply tests/integration/flows/test_adls_to_azure_sql.py]: You are right, changed!

**Review by Rafalz13 — APPROVED** [review]

**Pull request merged**